resources/data-provider.mjs

```diff
@@ -0,0 +1,165 @@
+// example url for local testing:
+// http://localhost:8080/?config=http://localhost:8080/resources/config.json
+import { defaultSuites } from "./tests.mjs";
+import { params } from "./shared/params.mjs";
+
+const DEFAULT_TAGS = ["all", "default", "experimental"];
+const ALLOWED_DOMAINS = {
+    "app.netlify.com": "/sites/webkit-speedometer-preview/",
```

Comment:
At the meeting today, we discussed changing this to restrict the hosts that are allowed to load config files at all (to netlify & localhost) **instead of** restricting the domains that tests are allowed to load from. This way it would be clear that browserbench.org never loads any external config/tests.